src/features/ExploreSlice.js

```diff
@@ -0,0 +1,46 @@
+const { createSlice, createAsyncThunk } = require("@reduxjs/toolkit");
+const { default: axios } = require("axios");
+
+
+const initialExploreState = {
+    videos: [],
+    loading: false,
+    error:"",
+}
+
+export const fetchVideos = createAsyncThunk("fetchVideos",
+    async (data,thunkAPI)=> {
+        try {
+            const { setVideoList } = data;
+            const resp = await axios.get('/api/videos');
+            setVideoList(resp?.data.videos)
+            return resp?.data;
+        } catch (error) {
+            return thunkAPI.rejectWithValue(error);
+}
+})
+
+
+const exploreSlice = createSlice({
+    name: "explore",
+    initialState: initialExploreState,
```

Comment:
```suggestion
   initialExploreState,
```

this will also work same

Reply:
It will not work in the same manner as the slice requires the parameter to have `initialState` so what Maruthi has done will be good to go.

Reply:
Oh. Thought that wouldn't work. Thankyou.